htdocs/core/class/html.formmail.class.php

```diff
@@ -692,26 +692,30 @@ public function get_form($addfileaction = 'addfile', $removefileaction = 'remove
 				} else {
 					// The free input of email
 					if (!empty($this->withtofree)) {
-						$out .= '<input class="minwidth200" id="sendto" name="sendto" value="'.(($this->withtofree && !is_numeric($this->withtofree)) ? $this->withtofree : (!is_array($this->withto) && !is_numeric($this->withto) ? (GETPOSTISSET("sendto") ? GETPOST("sendto") : $this->withto) : "")).'" />';
```

Comment:
I'm afraid the  "multiselectarray" with free entry introduces a big dependency on the multiselect that may be difficult to go back the day we will have to change the select library, also it will break usability for the mode "blind people".
So can you for the moment introduce a hidden option to use the new multiselect with free text for email choice only when this option is set. So we will keep code compatible for blind people and keep a code to go back with full features (free email and select) in case we have to change library where free entry is not possible.
I will see in future when we can switch by default to the new version...